GVFS/GVFS.Common/Prefetch/BlobPrefetcher.cs

```diff
@@ -129,6 +149,44 @@ public static bool TryLoadFileList(Enlistment enlistment, string filesInput, Lis
             return true;
         }
 
+        public static bool IsNoopPrefetch(
+            ITracer tracer,
+            FileBasedDictionary<string, string> lastPrefetchArgs,
+            string commitId,
+            List<string> files,
+            List<string> folders,
+            bool hydrateFilesAfterDownload)
+        {
+            if (lastPrefetchArgs != null &&
+                lastPrefetchArgs.TryGetValue(PrefetchArgs.CommitId, out string lastCommitId) &&
+                lastPrefetchArgs.TryGetValue(PrefetchArgs.Files, out string filesString) &&

```

Comment:
For consistency could you prepend "last" on the other out parameters as well? (e.g. lastFilesString, lastFoldersString, etc.)